packages/js/tests/components/SidebarCollapsibleTest.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import SidebarCollapsible from "../../src/components/SidebarCollapsible";
+import { fireEvent, render, screen } from "../test-utils";
+
+const defaultArgs = {
+	title: "Test title",
+	children: <div id="inner-div">Test children</div>,
```

Comment:
Love the tests! 🙌 
But where is your fancy new buttonId prop? 😁

Reply:
Added! 😁